app/v1/validation/fullReturn/AdjustedGroupInterestValidator.scala

```diff
@@ -137,3 +139,10 @@ trait AdjustedGroupInterestValidator extends BaseValidation {
     val path: JsPath = topPath \ "groupRatio"
     val value = Some(Json.toJson(groupRatio))
   }
+
+case class NegativeOrZeroQNGIEError(groupRatio: BigDecimal)(implicit topPath: JsPath) extends Validation {
+  val code = "QNGIE_NEGATIVE"
+  val errorMessage: String = "If group QNGIE is negative or zero, set group ratio to 100"
```

Comment:
Talk to Anita about an error message for this, but the error is simply that QNGIE can't be negative